scripts/build.sh

```diff
@@ -10,78 +10,36 @@ if [[ "x$version" == "x" ]]; then
   exit 1
 fi
 
-# Create a direcotry to save assets
-ASSETSDIR=release
-mkdir $ASSETSDIR
-
-function build_amd64_linux() {
-  DIR=pulsarctl-amd64-linux
-  mkdir ${DIR}
-  CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o pulsarctl -ldflags "-X github.com/streamnative/pulsarctl/pkg/pulsar.ReleaseVersion=Pulsarctl-Go-$version" .
-  mv pulsarctl ${DIR}
-  cp -r plugins ${DIR}
-  tar -czf ${DIR}.tar.gz ${DIR}
-  mv ${DIR}.tar.gz $ASSETSDIR
-}
+ROOT_DIR=`cd $(dirname $0)/../; pwd`
```

Comment:
I think use the `git rev-parse --show-toplevel` is better.

Reply:
`git` command will make it only run with a git repo. `dirname` can run it anywhere.